core/domain/training_job_result_domain_test.py

```diff
@@ -37,9 +37,9 @@ def test_validate_job_data_with_valid_model_does_not_raise_exception(self): # py
             job_id, algorithm_id, classifier_data)
         job_result.validate()
 
-    def test_validate_job_data_with_invalid_model_raises_exception(self):
-        """Ensure that validation checks raise exception when
-        an invalid classifier model is supplied.
+    def test_job_data_with_invalid_job_id_raises_exception_on_validtion(self):
```

Comment:
Optional, but you can also rearrange the test names as so:

``test_validate_with_invalid_job_id_raises_exception``

This keeps the format _test\_action\_with\_condition\_and\_condition\_has\_outcome_. Note that I think the 'job_data' bit is implied given the class name.

Reply:
Done